src/bundles/experiments.js

```diff
@@ -0,0 +1,122 @@
+import { createSelector } from 'redux-bundler'
+
+export const ACTIONS = {
+  EXP_TOGGLE_STARTED: 'EXPERIMENTS_TOGGLE_STARTED',
+  EXP_TOGGLE_FINISH: 'EXPERIMENTS_TOGGLE_FINISH',
+  EXP_TOGGLE_FAILED: 'EXPERIMENTS_TOGGLE_FAILED',
+  EXP_UPDATE_STATE: 'EXPERIMENTS_UPDATE_STATE'
+}
+
+const EXPERIMENTS = [
+  {
+    key: 'npmOnIpfs',
+    actionUrls: [
+      {
+        url: 'https://github.com/ipfs-shipyard/npm-on-ipfs',
+        key: 'readMoreUrl'
+      },
+      {
+        url: 'https://github.com/ipfs-shipyard/npm-on-ipfs/issues',
+        key: 'issueUrl'
+      },
+      {
+        url: 'https://github.com/ipfs-shipyard/npm-on-ipfs',
```

Comment:
We need a plan for how to gather feedback... We could create a "Feedback" issue, and have folks comment on it.  Won't scale if we  get tons of feedback, but I think we can deal with that problem when it happens.

Reply:
Should it be there, or on `ipfs-desktop` repository?

Reply:
would directing people to a new issue template be a viable option for this?

Reply:
Neat, what would it say? How would we distinguish it from raise an issue? Are issues a good place for general feedback? They kinda work for it, but nice as it may be, we don't want hundreds of issues that say "i love it" or "I hate it".

Of note, countly has a feedback / ratings plugin https://count.ly/plugins/ratings-and-feedback
though it feels a little gross to popup emojis. Any other options?